standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -138,15 +149,18 @@ private ReplChangeManager(Configuration conf) throws MetaException {
       if (!inited) {
         if (MetastoreConf.getBoolVar(conf, ConfVars.REPLCMENABLED)) {
           ReplChangeManager.enabled = true;
-          ReplChangeManager.cmroot = new Path(MetastoreConf.getVar(conf, ConfVars.REPLCMDIR));
           ReplChangeManager.conf = conf;
 
+          //Create default cm root
+          Path cmroot = new Path(MetastoreConf.getVar(conf, ConfVars.REPLCMDIR));
           FileSystem cmFs = cmroot.getFileSystem(conf);
           // Create cmroot with permission 700 if not exist
           if (!cmFs.exists(cmroot)) {
             cmFs.mkdirs(cmroot);
             cmFs.setPermission(cmroot, new FsPermission("700"));
           }
+          cmRootMapping.put(NO_ENCRYPTION, cmroot);
```

Comment:
This is for REPLCMDIR. This will always be unencrypted. Encryption cm directory we have separated. If there is any unencrypted path for which recycle is called, we will use this. Else we will create a cm for that zone